boards/arduino-zero/dist/openocd.cfg

```diff
@@ -1 +0,0 @@
-source [find target/at91samdXX.cfg]
```

Comment:
That's only valid for `samd2x` and `saml2x` - for saml1x there is `atsaml1x.cfg` and `samd5x` uses `atsame5x.cfg`.

I think we can have a common folder for samd21 based boards, there is a lot of copy & paste going on wrt. timer & clock setup.
So I'd say `common/sam0` is to broad but having a `common/sam21` would be nice.

Reply:
Oh I just realized this PR only touches samd21 based boards.
In that case this change is valid, but the folder should then really be called `samd21` instead of `sam0`.

Reply:
Ah not quite - some saml21 based boards have sneaked in.